packages/web/src/pages/upload-page/UploadPage.js

```diff
@@ -248,10 +248,6 @@ class Upload extends Component {
     // This will change once we introduce premium content UI
     // in the track upload flow.
     const tracks = [...this.state.tracks]
```

Comment:
I think we don't need this spread anymore? Also is comment outdated?

Reply:
good catch